tesseract_motion_planners/trajopt/src/trajopt_utils.cpp

```diff
@@ -192,7 +192,8 @@ trajopt::TermInfo::Ptr
 createSmoothVelocityTermInfo(int start_index, int end_index, int n_joints, double coeff, trajopt::TermType type)
 {
   if ((end_index - start_index) < 1)
-    throw std::runtime_error("TrajOpt JointVelTermInfo requires at least two states!");
+    throw std::runtime_error("TrajOpt JointVelTermInfo requires at least two states, you only have " +
+                             std::to_string(end_index - start_index + 1) + "!");
```

Comment:
Might be better to just print include the start and end index instead of the difference.

Reply:
Let me know how the new language sounds